src/gui/guiFormSpecMenu.cpp

```diff
@@ -1431,11 +1513,16 @@ void GUIFormSpecMenu::parseTabHeader(parserData* data, const std::string &elemen
 			Environment->setFocus(e);
 		}
 
-		e->setNotClipped(true);
+		auto style = getThemeForElement("tabheader", name);
+		e->setNotClipped(style.getBool(StyleSpec::NOCLIP, true));
 
 		for (const std::string &button : buttons) {
-			e->addTab(unescape_translate(unescape_string(
+			auto tab = e->addTab(unescape_translate(unescape_string(
 				utf8_to_wide(button))).c_str(), -1);
+			if (style.hasProperty(StyleSpec::BGCOLOR))
+				tab->setBackgroundColor(style.getColor(StyleSpec::BGCOLOR));
```

Comment:
This isn't documented because it doesn't actually work

![image](https://user-images.githubusercontent.com/2122943/54482457-c2347980-483b-11e9-9a59-7915b82f6658.png)